-- Drop PEP 639 license metadata until the FreeBSD pyproject toolchain handles it.
-- Remove the setuptools runtime dependency because the port only needs setuptools to build.

--- pyproject.toml.orig	2025-10-07 21:08:35 UTC
+++ pyproject.toml
@@ -14,8 +14,6 @@ readme = "README.rst"
 authors = [{ name = "pymc-devs", email = "pymc.devs@gmail.com" }]
 description = "Optimizing compiler for evaluating mathematical expressions on CPUs and GPUs."
 readme = "README.rst"
-license = "BSD-3-Clause"
-license-files = ["LICENSE.txt"]
 classifiers = [
     "Development Status :: 6 - Mature",
     "Intended Audience :: Education",
@@ -46,7 +44,6 @@ dependencies = [
     "differentiation",
 ]
 dependencies = [
-    "setuptools>=59.0.0",
     "scipy>=1,<2",
     "numpy>=2.0",
     "filelock>=3.15",
